<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JudgmentForm</class>
 <widget class="QWidget" name="JudgmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(222,232,235);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFriends">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAgeIndex">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFamily">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAge">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSin">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxVirtue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxTravel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxExercise">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSinIndex">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxVirtueIndex">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Friendship Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Family Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Sin Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Virtue Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Travel Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Exercise Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190,209,216);</string>
   </property>
   <property name="text">
    <string>Age Points</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
